install.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+
+# VARIABLES, CHANGE AS NEEDED
+GITHUB_REPO="https://github.com/bmFtZQ/edge-frfox.git"
+PROJECT_NAME=$(basename $GITHUB_REPO | cut -d '.' -f 1)
+TMP_DIR="${TMPDIR:-$(dirname $(mktemp))}"
+
+if [[ $OSTYPE == "darwin"* ]]; then
+  FIREFOX_DIR=$HOME/Library/Application\ Support/Firefox
```

Comment:
Use double quotes here for the path, also would probably be best if double quotes are used for all commands involving the profile or Firefox path due to the space in the path name.